rules/things-to-automate-stage-2/rule.md

```diff
@@ -17,10 +17,12 @@ redirects:
 
 ---
 
-Once you’ve identified the manual processes in [Stage 1](/things-to-measure), you can start looking at automation. The best tool for build and release automation is Azure DevOps.
+Once you’ve identified the manual processes in [Stage 1](/things-to-measure), you can start looking at automation. The two best tools for build and release automation are Github (public projects) and Azure DevOps (private projects).
```

Comment:
Are we advocating against GitHub Private Projects? How come?

Reply:
I wanted to give a recommendation for why to use one or the other. At this point GitHub has better public/open source projects so DevOps got the other one